test-harness/src/test/java/com/netflix/conductor/tests/integration/AbstractEndToEndTest.java

```diff
@@ -0,0 +1,23 @@
+package com.netflix.conductor.tests.integration;
+
+import com.netflix.conductor.common.metadata.tasks.TaskDef;
+import com.netflix.conductor.common.metadata.workflow.TaskType;
+import com.netflix.conductor.common.metadata.workflow.WorkflowTask;
+
+public class AbstractEndToEndTest {
+    protected static final String TASK_DEFINITION_PREFIX = "task_";
```

Comment:
**Question:** I wonder if in the future it would make sense to setup a shared field/getter that could be set during construction to help with the name spacing issues we have discussed?

Reply:
Good question. I am sure we can do something. I noticed there is a property called `workflow.namespace.prefix` that you can pass to the TestEnvironment. I played with it in the past when working on the integration tests. Will make a note for this.
In this case, this prefix is just used in order to avoid hardcoding `task_` over the code.

Reply:
In addition to my last comment and answering your question,  I think the namespace is something we can do in a near future modifying slightly the `TestEnvironment` class.